osu.Game.Rulesets.Catch.Tests/TestSceneFruitRandomness.cs

```diff
@@ -26,6 +26,8 @@ public TestSceneFruitRandomness()
             AddSliderStep("start time", 500, 600, 0, x =>
             {
                 drawableFruit.HitObject.StartTime = drawableBanana.HitObject.StartTime = x;
+                drawableFruit.RefreshStateTransforms();
+                drawableBanana.RefreshStateTransforms();
```

Comment:
I'd rather go through `drawableFruit.StartTimeBindable.Value = ...` than do this.

Reply:
I agree this test scene code looks a bit ugly, but I disagree going through `StartTimeBindable` to refresh the state. The only way a hit object start time gets changed in non-test environments is through the editor, so `DefaultsApplied` correctly updates the drawable for all real scenarios. Having the refresh through the `StartTimeBindable` is completely unnecessary.

Since this test is not in an `EditorBeatmap` so `DefaultsApplied` doesn't get invoked, the drawable must be refreshed manually. To add automatic refresh of the drawable state based on changes in the `StartTimeBindable` just for this one test, which isn't even about the change handling, is kind of absurd.